app/controllers/api/v1/tasks_controller.rb

```diff
@@ -34,6 +34,7 @@ def create
 
   def update
     task = Task.find(params[:id])
+    updateState task, params[:task][:state]
```

Comment:
just send state_event instead of state, it will do it automatically through update

Reply:
use rails snake_case notation in naming